### DISCUSSION TITLE
libfmt.so.8 dependency outdated on linux Arch/Manjaro

### DISCUSSION THREAD

**Discussion by Tonipt** [diasurgical/DevilutionX#5353] [Q&A]
devilutionx: error while loading shared libraries: libfmt.so.8: cannot open shared object file: No such file or directory.
For systems based on Arch Linux. Needed dependency for libfmt.so.9 library.

**Comment by Tonipt** [comment]
Ok. I recompile devilutionX on my system and now works! Sorry for bothering!

**Reply by AJenbo** [reply]
https://aur.archlinux.org/packages/devilutionx
